processors/base/writers/source.py

```diff
@@ -48,6 +49,21 @@ def write_source(conn, source):
         'terms_and_conditions_url': source.get('terms_and_conditions_url'),
     })
 
+    # Validate object
+    url_fields = ['url', 'terms_and_conditions_url']
+    failed_url_validation = [field for field in url_fields
+                             if not helpers.validate_remote_url(object[field])]
+    if failed_url_validation:
+        for field in failed_url_validation:
```

Comment:
Instead of printing multiple log messages, print only one passing the actual `failed_url_validation` array. It'll be converted to string anyway, and makes our logs less noisy.